src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-CSharp/.template.config/template.json

```diff
@@ -12,7 +12,6 @@
   "precedence": "3000",
   "identity": "Microsoft.Web.Empty.CSharp.2.1",
   "shortName": "web",
-  "thirdPartyNotices": "https://aka.ms/template-3pn",
```

Comment:
Some templates like Empty and WebAPI do not use any third party libraries and do not need the link. Safe to remove?